sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/DownloadBlobToOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Containers.ContainerRegistry.Specialized
+{
+    /// <summary>
+    /// Options to pass to the DownloadBlobTo method.
+    /// </summary>
+    public class DownloadBlobToOptions
+    {
+        private readonly int _maxChunkSize;
+
+        /// <summary>
+        /// Creates a new instance of DownloadBlobToOptions.
+        /// </summary>
+        /// <param name="maxChunkSize">The maximum size of chunk to transfer in a single request.</param>
+        public DownloadBlobToOptions(int maxChunkSize)
+        {
+            _maxChunkSize= maxChunkSize;
+        }
+
+        /// <summary>
+        /// The maximum size of chunk to transfer in a single request.
+        /// </summary>
+        public int MaxChunkSize => _maxChunkSize;
```

Comment:
nit: any reason for **max**ChunkSize ? Why not just `ChunkSize`?

Reply:
It was to be approximately consistent with storage, which has [MaximumTransferSize](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Common/src/StorageTransferOptions.cs#L29) - I assume this is because the final chunk will often be smaller than the maximum size.  Shall we go with MaximumTransferSize instead?  I kept `chunk` because it is closer to the Registry API, but since we're aligning on other names, perhaps we should go in at 100%.

Reply:
I've removed the options type from the public API.